apmpackage/apm/data_stream/service_interval_metrics/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -0,0 +1,22 @@
+---
+description: Pipeline for ingesting APM internal metrics.
+processors:
+  - pipeline:
+      name: observer_version
+  - pipeline:
+      name: observer_ids
+  - pipeline:
+      name: ecs_version
+  - pipeline:
+      name: user_agent
+  - pipeline:
+      name: process_ppid
+  - pipeline:
+      name: client_geoip
+  - remove:
+      field: _metric_descriptions
+      ignore_missing: true
+  - remove:
+      # Removed in 8.6.0
+      field: timeseries.instance
+      ignore_missing: true
```

Comment:
Do we really need all these pipeline processors? E.g. why the `process_ppid`, `user_agent` and `client_geoip`, or the two remove processors?

Reply:
good catch, we don't need them. I'll remove the ones that are not needed.